helloworld/build.gradle

```diff
@@ -25,7 +25,7 @@ apply plugin: 'scalaStyle'
 apply plugin: 'com.commercehub.gradle.plugin.avro'
 
 apply from: 'gradle/tests.gradle'
-apply from: 'gradle/spark.gradle'
+apply from: "${rootProject.projectDir}/../gradle/spark.gradle"
```

Comment:
we can consult git for the local repo path to eliminate the need for `..`.
```
def gitRootOS = new ByteArrayOutputStream()
gitRootOS.withStream { os ->
    exec {
        commandLine 'git', 'rev-parse', '--show-toplevel'
        standardOutput = os
    }
}
def gitRoot = gitRootOS.toString().trim()
apply from: "${gitRoot}/gradle/spark.gradle"
```

Reply:
@gerashegalov We could try that. However, it could fail if one downloaded a zip of the repo or if we published it as a package down the line. Should we add this as well perhaps ?
```
if ( "${gitRoot}".compareTo("fatal: not a git repository (or any of the parent directories): .git") == 1 ) {
    gitRoot= "${rootProject.projectDir}/.."
    } 
```

Reply:
this is a good and valid point. Short term we can use your proposal (probably better use [ExecResult.getExitValue](https://docs.gradle.org/current/javadoc/org/gradle/process/ExecResult.html#getExitValue--) to catch all kinds of git invocation problems. 

Longer term, this hello world project can be configured by the same parent project.

Reply:
@gerashegalov So do you suggest checking for gitRoot in this PR?

Reply:
@py-ranoid yes, can we try what you do here https://github.com/salesforce/TransmogrifAI/pull/254#discussion_r270288085 but in a more general fashion. Instead of just checking the singl error message "fatal: not a git..." can we check for non-zero ExecResult.getExitValue which should capture all kinds of problems when invoking the git command?